InventorySalesApi/Api/Data/AreProductsSalableResultInterface.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventorySalesApi\Api\Data;
+
+use Magento\Framework\Api\ExtensibleDataInterface;
+
+/**
+ * Represents result of service Magento\InventorySalesApi\Api\AreProductsSalableForRequestedQtyInterface::execute()
+ *
+ * @api
+ */
+interface AreProductsSalableResultInterface extends ExtensibleDataInterface
```

Comment:
I believe the name `ProductsSalableResultInterface` will be better here.

Reply:
@sidolov, renamed